library/x509_crt.c

```diff
@@ -2014,7 +2014,8 @@ static int x509_crt_find_parent_in(
 {
     int ret;
     mbedtls_x509_crt *parent, *fallback_parent;
-    int signature_is_good, fallback_signature_is_good;
+    int signature_is_good = 0;
+    int fallback_signature_is_good;
```

Comment:
The code might be easier to understand - and avoid unnecessary initialization - if we instead moved the code-block
```
        *r_parent = parent;
        *r_signature_is_good = signature_is_good;
```
to before the `break` in the loop.

Reply:
i'm ok with that.  Let me re-test and re-push